script/Arcueid/Day 7/07_00_ARC07_1C.txt

```diff
@@ -37,62 +41,68 @@ That's how it is. Nothing else exists inside that head of his other than sleep,
 [sha:b3993d52e09fe4edfc26f7845e68d21eede0a769]{
 -- Page 93, Offset 16799.
 -- むう、と苦虫を噛みつぶしたかの如く顔をしかめる有彦。
-Arihiko frowns like he bit down on a bitter bug.
+// 苦虫を噛み潰したよう en sayings? Bitter pill?
```

Comment:
Meh, you got the idiom here, can't really ask for more than that here. A "bitter pill to swallow" is like a difficult truth. Maybe "Arihiko frowns; must've been a bitter pill to swallow" but that's not quite right either.